test/python/transpiler/test_bip_mapping.py

```diff
@@ -267,3 +268,39 @@ def test_unconnected_qubit_subset(self):
         coupling = CouplingMap([(0, 1), (1, 3), (3, 2)])
         with self.assertRaises(TranspilerError):
             BIPMapping(coupling, qubit_subset=[0, 1, 2])(circuit)
+
+
+@unittest.skipUnless(HAS_CPLEX, "cplex is required to run the BIPMappingModel tests")
+@unittest.skipUnless(HAS_DOCPLEX, "docplex is required to run the BIPMappingModel tests")
+class TestBIPMappingModel(QiskitTestCase):
+    """Tests the BIPMappingModel pass."""
+
+    def test_objective_values(self):
+        """Test the optimal values are unchanged for each objective type."""
+        expected_dic = {
+            "depth": 1.02,
+            "error_rate": 0.4494397462045886,
+            "balanced": 0.4594397462045886,
```

Comment:
can you add a comment about what these magic values mean?
they are showing the optimal value found with that objective in mind, right? but they are not comparable to each other at all?
I think maybe a better test is to show that the circuit coming out of "depth" objective has a depth of D1 and error rate of E1, and the one coming out of "error_rate" objective has a depth of D2 and error of E2, and then it is easier to see D1 < D2 but E2 < E1.

Reply:
Updated as you suggested at bbe970e